### DISCUSSION TITLE
How to copy tags of an item and then paste the tags to another item?

### DISCUSSION THREAD

**Discussion by wuyulele** [windingwind/zotero-actions-tags#159] [Q&A] [answered]
Hi guys,

could someone provide a script to copy tags of a selected item into clipboard, and then paste the copied tags to another item? I guess this may need two sciprts or actions. Zutilo has the same function, but it is not compatible with  Zotero 7 yet.

Thank you very much!

**Comment by redleafnew** [comment, marked as answer]
https://github.com/windingwind/zotero-actions-tags/discussions/194

**Reply by wuyulele** [reply]
thank you very much!
